packages/discovery-react-components/src/components/DocumentPreview/components/PdfViewerWithHighlight/PdfViewerWithHighlight.tsx

```diff
@@ -99,32 +101,51 @@ const PdfViewerWithHighlight = forwardRef<any, Props>(
 
     const setCurrentErrMsgFromPdfConst = useIsPfdError(_isPdfRenderError, setIsPdfRenderError);
 
+    // TODO: Dynamically create a style for every category. Match color of category
+    const styleColor = Object.values(facetInfoMap || {})
+      .map(facetInfo => {
+        return `
+        .category_${facetInfo.facetId}.mf-highlight {
+          background: ${facetInfo.color};
+          border: 2px solid ${facetInfo.color};
+        }
+        .category_${facetInfo.facetId}.mf-active {
+          background: unset;
+          opacity: 100%;
+          border: 2px solid ${facetInfo.color};
```

Comment:
Both have this `border` set, so that should just be part of a `.category_${facetInfo.facetId}` rule.

Reply:
Also, some of these styles just apply if (1) it is a highlight and (2) if it is active. Those can be pulled out into the stylesheet file and don't need to be defined dynamically. The only things that need to be set dynamically here are the styles using `facetInfo`.

Reply:
I am trying to to refactor the styles, however, in this version the specificity of the `active` is lower than the `highlight`. I welcome css suggests to correct it

![image](https://github.com/watson-developer-cloud/discovery-components/assets/45341518/66f3448a-15da-4431-b195-3777fbdde94e)